lesson_09/oop/oop_app/src/test/java/com/codedifferently/lesson9/JadasLibraryTest/JadaslibraryTest.java

```diff
@@ -0,0 +1,61 @@
+package com.codedifferently.lesson9.JadasLibraryTest;
```

Comment:
This package should be the same as the package of the class you're testing. You probably noticed that you couldn't find your `Library` class, which I guessed from the fact that you copied the `Library` class into the test package.